Sell/Product/Models/ProductModifierItem.swift

```diff
@@ -36,7 +36,7 @@ extension ProductModifierItem {
     init?(response: ApolloType.FragmentProductModifierItem?) {
         guard let response = response else { return nil }
         id = response.id
-        status = response.status?.rawValue
+        status = ProductModifierItemStatus(rawValue: response.status?.rawValue ?? "")
```

Comment:
Decode response to the raw String, and then converts to the enum. When a new enum case is added by backend, the decoding will not fail.